spec/views/idv/shared/_document_capture.html.erb_spec.rb

```diff
@@ -38,6 +39,8 @@
     end
   end
 
+  assign(:presenter, presenter)
```

Comment:
I wouldn't expect an `assign` statement to occur outside a `before` or `it` block like this. Should it be moved into the `before` above?

Reply:
Heh yeah I caught this and fixed it in a previous commit